scripts/lumo_test.cljs

```diff
@@ -8,4 +9,36 @@
   (when-not (cljs.test/successful? m)
     (lumo/exit 1)))
 
+(defmethod cljs.test/report [:cljs.test/default :error] [m]
```

Comment:
does adding the code below help you debug what's going on in tests?

Reply:
Yep this is a better reporter basically, using this at work too, can take it away and add it in another PR as well if you want